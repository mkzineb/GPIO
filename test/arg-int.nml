let proc = "test"
let bit_order = "uppermost"
let gliss_isize = "8"
let* N: int(8) = 0

reg PC[1, card(32)] pc = 1
mem M[32, card(8)]

op instruction(x: all)
  syntax = x.syntax
  image = x.image
  action = { x.action ; }
  
op all = i

op i()
	image = "0000 0000"
	syntax = "i"
	action = { }

