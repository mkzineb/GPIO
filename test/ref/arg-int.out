let bit_order = "uppermost"	gliss-line-info = arg-int.nml:2

let proc = "test"	gliss-line-info = arg-int.nml:1

param __IADDR (card(32))
op instruction (x : all)
	action = {
		x.action;
	}
	image = x.image
	syntax = x.syntax
	gliss-line-info = arg-int.nml:9

let gliss_isize = "8"	gliss-line-info = arg-int.nml:3

op all = i	gliss-line-info = arg-int.nml:14

let N = 1L	gliss-line-info = arg-int.nml:4

op i ()
	action = {
		 <NOP>;
	}
	syntax = const(string, "i")
	image = const(string, "0000 0000")
	gliss-line-info = arg-int.nml:16

param __ISIZE (card(32))
mem M [32, card(8)] 	gliss-line-info = arg-int.nml:7

reg PC [1, card(32)] 	pc = const(card(32), 1L)
 	gliss-line-info = arg-int.nml:6

